KillTeam/Views/VersionPage.xaml

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d"
+             Title="Version"
+             x:Class="KillTeam.Views.VersionPage">
+    <ContentPage.Content>
+        <StackLayout>
+            <StackLayout
+              VerticalOptions="CenterAndExpand"
+              HorizontalOptions="FillAndExpand">
+                <Label Text="{Binding AppVersion}" HorizontalTextAlignment="Center" />
+                <Label Text="{Binding RulesVersion}" HorizontalTextAlignment="Center"/>

```

Comment:
Should we display a label before the version ? 

e.g.

App : [AppVersion]
Db : [DbVersion]

Reply:
We could, but I can't make the translate Resx files build, so I won't be able to add translations.